src/index.html

```diff
@@ -22,6 +22,397 @@
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars__star--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews card__reviews--count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--name">Price:</p>
+          <p class="card__price card__price--number">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars__star--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews card__reviews--count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--name">Price:</p>
+          <p class="card__price card__price--number">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars__star--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews card__reviews--count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--name">Price:</p>
+          <p class="card__price card__price--number">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div
```

Comment:
Use the article tag for the card
```suggestion
      <article
```

Reply:
fixed